packages/autorest.python/autorest/multiapi/serializers/import_serializer.py

```diff
@@ -4,17 +4,39 @@
 # license information.
 # --------------------------------------------------------------------------
 from copy import deepcopy
-from typing import Dict, Set, Optional, List
+from typing import Dict, Set, Optional, List, Tuple, Union
```

Comment:
Changes in `multiapi/models/imports.py` and `multiapi/models/import_serializer.py` are for supporting `import submodules by python versions` in multiapi files.

This feature is not used in current generation result (because we don't have type annotation on multiapi files), so it's only reserved for future..